person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = object_id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
```

Comment:
- [ ] Kindly make this method private method since it is mentioned in the requirements to make it a private method 🙏🏻🚀🔥

![image](https://github.com/Movgto/oop-school-library/assets/58553711/163ebc39-6309-4870-88b4-dfd49996cb70)